shortfin/pyproject.toml

```diff
@@ -34,6 +34,13 @@ dynamic = ["version"]
 Repository = "https://github.com/nod-ai/SHARK-Platform"
 Documentation = "https://shortfin.readthedocs.io/en/latest/"
 
+[project.optional-dependencies]
+apps = [
+  "transformers",
+  "dataclasses-json",
+  "pillow",
+]
```

Comment:
This will allow us to simplify the setup instructions https://github.com/nod-ai/SHARK-Platform/blob/3e7429544fd81938e2b1c0d8fbb098b1c871ecf7/shortfin/python/shortfin_apps/sd/README.md?plain=1#L6-L15

to just one of these:
```bash
pip install shortfin[apps]
pip install shark-ai[apps]
```